library/Mockery/CountValidator/Exact.php

```diff
@@ -33,12 +33,15 @@ class Exact extends CountValidatorAbstract
     public function validate($n)
     {
         if ($this->_limit !== $n) {
+            $because = $this->_expectation->getExceptionMessage();
+
             $exception = new Mockery\Exception\InvalidCountException(
                 'Method ' . (string) $this->_expectation
                 . ' from ' . $this->_expectation->getMock()->mockery_getName()
                 . ' should be called' . PHP_EOL
                 . ' exactly ' . $this->_limit . ' times but called ' . $n
                 . ' times.'
+                . ($because ? ' Because ' . $this->_expectation->getExceptionMessage() : '')
```

Comment:
Could do :) This is consistent with the method call but Reason makes sense also, I'll let you call it...